vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+---
+title: "Using Chromosome Bands as Categories"
+author: 
+   - name: D. Sarkar
+   - name: S. Falcon
+   - name: R. Gentleman
+   - name: Kemmy MO Jones
+     affiliation: [ Chromosome Bands Vignette translation from Sweave to R 
+     Markdown / HTML ]
+date: "`r format(Sys.Date(), '%B %d, %Y')`"
+package: Category
+vignette: >
+  %\VignetteIndexEntry{Using Chromosome Bands as Categories}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+             toc: false
+bibliography: 
+    - cat.bib
```

Comment:
Combine lines 19 and 20 removing the `-`.

Reply:
it is the syntax according to YAML documentation you gave us, When a document is extending another it must be properly indented else it won't compile, I tried it that way never worked.